local-dev/5.6/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN curl -L phpunit https://phar.phpunit.de/phpunit-5.phar > /tmp/phpunit.phar \
     && mv /tmp/phpunit.phar /usr/local/bin/phpunit
 
 # nvm environment variables
-ENV NODE_VERSION 10.12.0
+ENV NODE_VERSION 11.0.0
```

Comment:
One idea, I believe nvm accepts latest as the node version.

Reply:
If the goal is to always be on the latest, one concern is making sure the Grunt and Gulp versions support the latest Node versions.

Reply:
I had intentionally not done latest as it got us into problems at UF. That said, for alpha the hardcoded version still feels appropriate. Thoughts?